src/clickonce/MageCLI/Application.resx

```diff
@@ -270,6 +271,13 @@ Options:
       Specifies if the deployment manifest will include the deployment 
       provider URL.
       
+  -MapFileExtensions &lt;true|false&gt; -mfe
```

Comment:
One minor issue before I approve this PR. `-MapFileExtentions` option should be listed right above `-MinVersion` as options are listed alphabetically. This change will produce new XLF files.